app/lib/handover/handover_calculation.rb

```diff
@@ -14,10 +14,12 @@ def calculate_handover_date(sentence_start_date:,
         [earliest_release_date - 15.months, :early_allocation]
       elsif is_indeterminate
         [earliest_release_date - 12.months, in_open_conditions ? :indeterminate_open : :indeterminate]
+      elsif sentence_start_date + 10.months >= earliest_release_date
+        [nil, :determinate_short]
+      elsif is_determinate_parole
+        [earliest_release_date - 12.months, :determinate_parole]
       else
-        calculate_determinate_handover_date(sentence_start_date: sentence_start_date,
-                                            earliest_release_date: earliest_release_date,
-                                            is_determinate_parole: is_determinate_parole)
+        [earliest_release_date - 8.months - 14.days, :determinate]
       end
```

Comment:
Inlined the `calculate_determinate_handover_date` method as it was harder to understand that way. And it was not used elsewhere.